sqlalchemy_ibmi/base.py

```diff
@@ -205,249 +245,245 @@ class XML(sa_types.Text):
-            return "DECIMAL(%(precision)s, %(scale)s)" % {
-                            'precision': type_.precision, 'scale': type_.scale}
-
+        return "DECIMAL(%(precision)s, %(scale)s)" % {
+            'precision': type_.precision, 'scale': type_.scale}
 
-    def visit_numeric(self, type_):
-        return self.visit_DECIMAL(type_)
+    def visit_numeric(self, type_, **_):
+        return self.visit_DECIMAL(type_, )
 
-    def visit_datetime(self, type_):
-        return self.visit_TIMESTAMP(type_)
+    def visit_datetime(self, type_, **_):
+        return self.visit_TIMESTAMP(type_, )
 
-    def visit_date(self, type_):
-        return self.visit_DATE(type_)
+    def visit_date(self, type_, **_):
+        return self.visit_DATE(type_, )
 
-    def visit_time(self, type_):
-        return self.visit_TIME(type_)
+    def visit_time(self, type_, **_):
+        return self.visit_TIME(type_, )
 
-    def visit_integer(self, type_):
-        return self.visit_INT(type_)
+    def visit_integer(self, type_, **_):
+        return self.visit_INTEGER(type_)
 
-    def visit_boolean(self, type_):
-        return self.visit_SMALLINT(type_)
+    def visit_boolean(self, type_, **_):
+        return self.visit_SMALLINT(type_, )
 
-    def visit_float(self, type_):
-        return self.visit_FLOAT(type_)
+    def visit_float(self, type_, **_):
+        return self.visit_FLOAT(type_, )
 
-    def visit_unicode(self, type_):
-        return self.visit_VARGRAPHIC(type_)
+    def visit_unicode(self, type_, **_):
+        return self.visit_vargraphic(type_)
 
-    def visit_unicode_text(self, type_):
-        return self.visit_LONGVARGRAPHIC(type_)
+    def visit_unicode_text(self, type_, **_):
+        return self.visit_longvargraphic(type_)
 
-    def visit_string(self, type_):
-        return self.visit_VARCHAR(type_)
+    def visit_string(self, type_, **_):
+        return self.visit_VARCHAR(type_, )
 
-    def visit_TEXT(self, type_):
-        return self.visit_CLOB(type_)
+    def visit_TEXT(self, type_, **_):
+        return self.visit_CLOB(type_, )
 
-    def visit_large_binary(self, type_):
-        return self.visit_BLOB(type_)
+    def visit_large_binary(self, type_, **_):
+        return self.visit_BLOB(type_, )
 
 
 class DB2Compiler(compiler.SQLCompiler):
+    """ Compiler for DB2 """
+    def visit_empty_set_expr(self, element_types):
+        pass
+
+    def update_from_clause(self, update_stmt, from_table, extra_froms, from_hints, **kw):
+        pass
 
-    if SA_Version < [0, 9]:
+    def delete_extra_from_clause(self, update_stmt, from_table, extra_froms, from_hints, **kw):
+        pass
+
+    if SA_VERSION < [0, 9]:
         def visit_false(self, expr, **kw):
             return '0'
-            
+
         def visit_true(self, expr, **kw):
             return '1'
-            
+
     def get_cte_preamble(self, recursive):
         return "WITH"
-        
-    def visit_now_func(self, fn, **kw):
+
+    def visitNowFunc(self):
```

Comment:
This doesn't fit in the Python naming convention. Why did it change?

Reply:
This was an issue that pylint was complaining about.